wdio/tests/request-blocking.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { usersResponseSelector, usersSelector, expectedURL } from '../data/test-data.json';
+import { expect } from 'chai';
+
+describe('Request Blocking', () => {
+  before(async () => {
+    await browser.url('/');
+  });
+
+  describe('Blocking the request', () => {
+    before(async () => {
+      const getUsers = $(`${usersSelector}`);
+      browser.setupInterceptor();
+
+      const mockResponse = await browser.mock(`**${expectedURL}`, { method: 'get' });
+      mockResponse.abort('Aborted');
+      await getUsers.click();
+    });
+
+    it("should return no requests", async () => {
```

Comment:
blocking the request should return no requests?
Please explain this to me :]

Reply:
If the `mockResponse.abort('Aborted')` is called then the `await browser.getRequest(0)` returns with the given error message: `Could not find request with index 0`.

However, I removed this test, because this is not in the example file either, so our files should be identical.